.github/workflows/publish.yaml

```
@@ -75,8 +75,10 @@ jobs:
       - uses: actions/setup-go@v4
         with:
           go-version: '^1.20'
-      - name: make build
-        run: make build
+      - run: make build
+      - run: gh release create ${{ github.ref_name }} ./bin/replicated --generate-notes
```

looks like we can leave this empty if we want since we don't intend for users to use the sdk binary directly ([docs](https://cli.github.com/manual/gh_release_create))
```suggestion
      - run: gh release create ${{ github.ref_name }} --generate-notes
```